### PR TITLE
6098 operator status indicator v11 improvements

### LINKED ISSUE #6098: Operator Status Indicator improvements
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by scottbell**
Closes #6098  #6136

### Describe your changes:
Adds a "Clear Poll" button and an operator status table to the poll response indicator.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Reviewer has tested changes by following the provided instructions?
* [x] Changes appear not to be breaking changes?
* [x] Appropriate automated tests included?
* [x] Code style and in-line documentation are appropriate?
* [x] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -81,6 +81,8 @@
         openmct.install(openmct.plugins.example.EventGeneratorPlugin());
         openmct.install(openmct.plugins.example.ExampleImagery());
         openmct.install(openmct.plugins.example.ExampleTags());
+        openmct.install(openmct.plugins.example.ExampleUser());
```
Comment (scottbell) [inline index.html:84]: not sure if we want to include these, but it helps for testing!

Comment (michaelrogers) [reply index.html:84]: If we revert the install from index, the e2e tests will need to be updated to inject the plugins again.

**Review by unlikelyzero — APPROVED** [review]
Great work:)

**Comment by unlikelyzero** [comment]
Updated to include https://github.com/nasa/openmct/issues/6136

**Review by akhenry — CHANGES_REQUESTED** [review]
Looks great, thanks! Just two really minor suggestions.

**Comment by akhenry** [comment]
Have tested this and everything looks good. Just one weird issue with a tooltip when I hover over the blank Poll Question input in the FD indicator. I see the text "Go for egress?". Presumably placeholder text leftover from testing?

Screen Shot 2023-01-19 at 7 14 33 AM

**Comment by scottbell** [comment]
> Have tested this and everything looks good. Just one weird issue with a tooltip when I hover over the blank Poll Question input in the FD indicator. I see the text "Go for egress?". Presumably placeholder text leftover from testing?
> 
> Screen Shot 2023-01-19 at 7 14 33 AM

@akhenry Hmm, I'm not able to reproduce this locally. Do you perchance know how you were able to get the text?

**Comment by akhenry** [comment]
One more observation, the operator status indicator (ie. the "response" indicator) is still not appearing if there is no poll question set. But maybe this is actually desirable? @charlesh88 ? It's just taking up space if there is nothing to respond to.

**Comment by akhenry** [comment]
> > Have tested this and everything looks good. Just one weird issue with a tooltip when I hover over the blank Poll Question input in the FD indicator. I see the text "Go for egress?". Presumably placeholder text leftover from testing?
> > Screen Shot 2023-01-19 at 7 14 33 AM
> 
> @akhenry Hmm, I'm not able to reproduce this locally. Do you perchance know how you were able to get the text?

So it's not actually on hover (my bad), it's when I click on the empty input. 

Hovering over the subsequent tooltip then pre-populates the field with the same text (see screenshot)

Screen Shot 2023-01-19 at 7 28 46 AM

**Comment by scottbell** [comment]
@akhenry Here's me clicking on an empty input in vain 😀

https://user-images.githubusercontent.com/9853862/213484617-7f920eba-2052-453e-ace3-83f066b3aeb9.mov

**Comment by akhenry** [comment]
> > > Have tested this and everything looks good. Just one weird issue with a tooltip when I hover over the blank Poll Question input in the FD indicator. I see the text "Go for egress?". Presumably placeholder text leftover from testing?
> > > Screen Shot 2023-01-19 at 7 14 33 AM
> > 
> > 
> > @akhenry Hmm, I'm not able to reproduce this locally. Do you perchance know how you were able to get the text?
> 
> So it's not actually on hover (my bad), it's when I click on the empty input.
> 
> Hovering over the subsequent tooltip then pre-populates the field with the same text (see screenshot)
> 
> Screen Shot 2023-01-19 at 7 28 46 AM

As it transpires this was some Chrome autofill weirdness. At some point Chrome had apparently decided that "Go for egress?" was part of an address so it was giving what it thought was a useful context-specific autofill suggestion. Even clearing all my browsing data and using incognito did not clear it out. I eventually resolved it by just disabling autofill for addresses completely.

**Comment by akhenry** [comment]
@scottbell @michaelrogers Thank you for all of your hard work on this.

**Review by akhenry — APPROVED** [review]

**Pull request merged**